svc/challenge/repository/sql/migrations/20210812114010-create_attempts_table.sql

```diff
@@ -0,0 +1,12 @@
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS attempts (
+    user_id uuid NOT NULL,
+    episode_id uuid NOT NULL,
+    question_id uuid NOT NULL,
+    answer_id uuid DEFAULT NULL,
+    valid BOOLEAN DEFAULT FALSE,
+    created_at TIMESTAMP DEFAULT NOW(),
+    PRIMARY KEY (user_id, episode_id, question_id)
```

Comment:
@dmitrymomot  
if question_id is primary_key, we can't store the second attempt to  answer to question, isn't it?